### PR TITLE
WFLY-16128 [allowlist/blocklist] Problematic Language usage deprecation and replacement in testsuite

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://issues.redhat.com/browse/WFLY-16128

### CODE REVIEW

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks, @parsharma. I'm not resolving WFLY-16128 as I don't know whether this PR covers all the intended work on that JIRA. So I'll leave it up to you or @yersan to resolve if it should be resolved.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public static JavaArchive createArchive() {
     private DeserializationMessagingBean bean;
 
     @Test
-    public void testDeserializationBlackList() throws NamingException {
+    public void testDeserializationBlockList() throws NamingException {
         // UUID is black listed, any other Serializable must be deserialized.
```
Comment (yersan) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java:100]: Hi @parsharma , would you mind adding a new commit to fixing this comment too?

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java [hunk 2]
```diff
@@ -108,7 +108,7 @@ public void testDeserializationBlackList() throws NamingException {
     }
 
     @Test
-    public void testDeserializationBlackListFromRegularConnectionFactory() throws NamingException {
+    public void testDeserializationBlockListFromRegularConnectionFactory() throws NamingException {
         // all classes are black listed
```
Comment (yersan) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java:112]: Hi @parsharma , would you mind adding a new commit to fixing this comment too?

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java [hunk 3]
```diff
@@ -120,7 +120,7 @@ public void testDeserializationBlackListFromRegularConnectionFactory() throws Na
     }
 
     @Test
-    public void testDeserializationWhiteList() throws NamingException {
+    public void testDeserializationAllowList() throws NamingException {
         // UUID is white listed, any other Serializable must not be deserialized.
```
Comment (yersan) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java:124]: Hi @parsharma , would you mind adding a new commit to fixing this comment too?

**Review by yersan — COMMENTED** [review]
I've just found some comments with deprecated language on the test suite. @parsharma You can create an additional commit to tix those by using the same Jira.

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java (hunk 1 above)
Comment (parsharma) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/security/DeserializationBlockListTestCase.java:100]: Done https://github.com/wildfly/wildfly/pull/15443

**Pull request merged**